lib/newgistics/response_handlers/update_shipment_contents.rb

```diff
@@ -0,0 +1,25 @@
+module Newgistics
+  module ResponseHandlers
+    class UpdateShipmentContents
+      attr_reader :shipment
+
+      def initialize(shipment)
+        @shipment = shipment
+      end
+
+      def handle(response)
+        PostErrors.new(shipment).handle(response)
+        if shipment.errors.empty?
+          handle_successful_response(response)
+        end
+      end
+
+      private
+
+      def handle_successful_response(response)
+        xml = Nokogiri::XML(response.body)
+        shipment.success = xml.css('success').children.first.text
```

Comment:
I think we can re-write this like:

```ruby
shipment.success = xml.at_css('success').text
```

Also does Virtus parse `"true"` and `"false"` correctly to Booleans?

Reply:
yes Virtus will correctly parse `"true"` and `"yes"` to `true`and `"false"` and `"no"` to `false`